app/assets/javascripts/test/reducers/flycam_reducer.spec.js

```diff
@@ -155,7 +155,7 @@ test("Flycam should move in ortho mode", t => {
 });
 
 test("Flycam should move in ortho mode with dynamicSpaceDirection", t => {
-  let newState = FlycamReducer(initialState, FlycamActions.setRotationAction([0, 0, -2]));
```

Comment:
The test caption suggests that `dynamicSpaceDirection` is tested, but the tests didn't fail? Is it easy to add a new test which covers the scenario that you are fixing?